openid-connect-server/src/test/java/org/mitre/oauth2/service/impl/TestDefaultOAuth2AuthorizationCodeService

```diff
@@ -0,0 +1,53 @@
+package org.mitre.oauth2.service.impl;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+import org.junit.Test;
+import org.mitre.oauth2.model.AuthenticationHolderEntity;
+import org.mitre.oauth2.model.AuthorizationCodeEntity;
+import org.mitre.oauth2.repository.AuthorizationCodeRepository;
+import org.mitre.oauth2.repository.impl.JpaAuthenticationHolderRepository;
+import org.mitre.oauth2.repository.impl.JpaAuthorizationCodeRepository;
+import org.mockito.Matchers;
+import org.mockito.Mockito;
+import org.springframework.security.oauth2.provider.OAuth2Authentication;
+
+public class TestDefaultOAuth2AuthorizationCodeService {
+
+	@Test
+	public void testConsumeAuthorizationCode()
+	{
+
+DefaultOAuth2AuthorizationCodeService defaultAuth = new DefaultOAuth2AuthorizationCodeService();
+AuthenticationHolderEntity authHolder = Mockito.mock(AuthenticationHolderEntity.class);
```

Comment:
these mocks can be made using `@Mock` annotations and injected automatically into the service instead of being set directly; this is the preferred method as this mimics what Spring does at runtime